net/tayga/src/etc/rc.d/opnsense-tayga

```diff
@@ -21,6 +22,17 @@ command_args="-p ${pidfile}"
 
 [ -z "$tayga_enable" ] && tayga_enable="NO"
 
+tayga_start()
+{
+        echo "starting tayga"
+        ${command} ${command_args}
+        sleep 1
+        ifconfig nat64 inet ${tayga_v4destination}/32 ${tayga_v4address}
+        ifconfig nat64 inet6 ${tayga_v6address}/128
```

Comment:
The nat64 interface address must not be the `tayga_v6address`, but a separate address. Something like `tayga_v6tunaddress`. We need an additional input field for this in the UI.